consul-enterpise-admin-partitions-eks/consul_enterprise/cleanup/cleanup-load-balancers.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -e
+
+VPC_ID=${VPC_ID}
+ELASTIC_LOAD_BALANCERS=$(aws elb describe-load-balancers)
+
+# Security group description auto-generated by Kubernetes, and not created or managed by terraform
```

Comment:
AWS commands are failing due to no region being specified.

![image](https://user-images.githubusercontent.com/29551334/150573884-2a41737d-ba14-48dc-812a-c85aaea21cbf.png)